pkg/web/static/styles/output.css

```diff
@@ -3604,6 +3612,15 @@ body main {
   display: inline;
 }
 
+/* Set a media query for the transaction list table */
+
+@media (max-width: 1536px) {
```

Comment:
Manually setting the overflow here prevents scrollbars from appearing on ultra wide screens. `overflow-y` was also specified because scrollbars appeared along the y-axis though they weren't necessary. I'm pretty sure this is because of how the menu items appear once clicked.